CONTRIBUTING.md

```diff
@@ -1,35 +1,105 @@
+## Combine permits into one topic
 
-- Open [config/index.ts](apps/client/src/config/index.ts) in your editor of choice
-- Duplicate an existing object under `const topics` and edit these values: `slug`, `sttrFile`, `text` and `intro`
-  - `slug` = The part of our app URL that identifies which permit-check to load (`dakraam-plaatsen` will be `https://vergunningcheck.amsterdam.nl/dakraam-plaatsen`)
-  - `sttrFile` = The name of the (JSON :roll_eyes: ) file that holds all the question and route logic. An actual STTR file (XML :roll_eyes: ) is parsed and changed into JSON to be accessed more easily. 
-  - `text` = This is part that holds specific texts for each permit-check
-  - `intro` = The name of the component that has all texts on the Intro page (
-- Duplicate an existing Intro component file in `apps/client/src/intros`, change the content to your wishes and name it **exactly** as you named it in the previous step `intro`
+If you want to combine 2 or more permits in 1 checker, we need to configure a topic in our imtr-build config.
+
+- Follow the install steps in [./apps/imtr/README.md](./scripts/sttimtrr_build/README.md)
+- First make sure you update your permits so we have all the files we need. See the section above.
+- List the permit-id's you want to combine in [./apps/imtr/src/config.ts](./apps/imtr/src/config.ts) under `topics`.
+  - To find the permit-ids you can use in the `./apps/client/public/imtr/list.source.json`.
+  - You have to give your topic a slug as the key. This slug is used in the url by the client-application.
+- Then run `npm run imtr`.
+- Open the app, your checker should be visible in the list.
 
+A short note about combining permits. Often multiple permits in one checker include the same questions, eg. the monument-status question. We don't want to ask the same question twice so we dedupe them. Make sure the `herbruikbaarId` field is the same in the IMTR files. We use the key `uuid` in the `Question` model.
 
-@Andre, can you add the new steps how to generate a matching JSON file to the `sttrFile` key?
+## Configure a topic in the client-app
 
-_Text to be extended..._
+If you want to tweak texts, the name or the intro of a checker, you need to add some configuration in the client-application.
+
+- Open [config/index.ts](apps/client/src/config/index.ts) in your editor of choice
+- Duplicate an existing object under `const topics` and edit these values: `slug`, `text` and `intro`
+  - `slug` = The part of our app URL that identifies which permit-checker to load (`dakraam-plaatsen` will be `https://vergunningcheck.amsterdam.nl/dakraam-plaatsen`)
+  - `text` = This is part that holds specific texts for each permit-checker
+  - `intro` = The name of the component that has all texts on the Intro page (
+- Duplicate an existing Intro component file in `apps/client/src/intros`, change the content to your wishes and name it **exactly** as you named it in the previous step `intro`
```

Comment:
I miss some information about `hasImtr` here.

Reply:
I will move this documentation to the definition of `export type Topic` where it's actually more visible/usable for devs. Will describe all the props.

Reply:
![Schermafbeelding 2020-10-05 om 16 48 12](https://user-images.githubusercontent.com/1159693/95094797-96812900-072a-11eb-9984-eedca606432d.png)